### ISSUE TITLE
Improve EFB light theme "secondary" colors

### ISSUE THREAD

**Issue by alepouna** [flybywiresim/aircraft#8649]
### Aircraft Version

Development

### Description

A user posted a screenshot in the A32NX-support on Discord using the light theme on the EFB, it seems the color for the text is very close to the white background, and probably in a daylight condition (screenshot is in a night-time flight) is going to be almost invisible. 

![image](https://github.com/flybywiresim/aircraft/assets/98479040/6470dd15-1a11-4897-9d00-4ab36995d559)

Additionally for consideration to be other text like the PAX one on the pload page
![image](https://github.com/flybywiresim/aircraft/assets/98479040/5d9cd99a-151a-46b3-8442-91bfa4708c13)

**Comment by 2hwk** [comment]
Yes light mode has been a bit neglected. Something that needs more thought for flyPad OS 4.0 design overhaul.
